### PR TITLE
Allow lmdb mmap size to be set from HC_LMDB_SIZE

### PR DESCRIPTION
**Pull request by freesig**
Allow lmdb mmap size to be set from HC_LMDB_SIZE, size is in bytes and is automatically converted to a multiple of the pages size

### CODE REVIEW

**Comment by freesig** [comment]
We will need to rebase this onto develop before actually merging it

File: crates/test_utils/wasm/build.rs [hunk 1]
```diff
@@ -2,8 +2,8 @@
 //use std::process::Stdio;
 
 fn main() {
-    return;
-/*    let should_build = std::env::var_os("CARGO_FEATURE_BUILD").is_some();
+    // return;
+    /*    let should_build = std::env::var_os("CARGO_FEATURE_BUILD").is_some();
```
Comment (maackle) [inline crates/test_utils/wasm/build.rs:6]: not part of this PR but are we even using this build file anymore?

**Review by maackle — APPROVED** [review]
